src/machines/java/com/enderio/machines/common/block/TravelAnchorBlock.java

```diff
@@ -59,4 +62,12 @@ private static PlayerSneakEntry getLastSneakEntry(ServerPlayer player){
     }
 
     private record PlayerSneakEntry(boolean isSneaking, int atTime){}
+
+    @Override
+    public void onRemove(BlockState state, Level level, BlockPos pos, BlockState newState, boolean movedByPiston) {
```

Comment:
This method never gets called on the client if the travel anchor was broken by another player in a different dimension. If the client enters the dimension and places the travel anchor, leaves, and another player breaks the anchor, the client will still see the travel anchor when they return to the original dimension.

Reply:
If the block is actually removed on the server, this method should also be called on the client?